test/sasdataloader/test/utest_abs_reader.py

```diff
@@ -95,6 +100,7 @@ def test_igor_checkdata(self):
             Data1D defaults changed. Otherwise the
             tests won't pass
         """
+        self.data = Loader().load("MAR07232_rest.ASC")
```

Comment:
This is why I wrote my other suggestion to compare Loader.load to Reader.read. I actually wrote that one after this suggestion, but my full review came out in code order, not time order.